db/migrate/20240410210755_add_check_in_and_check_out_to_accomodations.rb

```diff
@@ -1,6 +1,6 @@
 class AddCheckInAndCheckOutToAccomodations < ActiveRecord::Migration[7.1]
   def change
-    add_column :accomodations, :check_in, :time
-    add_column :accomodations, :check_out, :time
+    add_column :accomodations, :check_in, :datetime
+    add_column :accomodations, :check_out, :datetime
   end
```

Comment:
Check in and check out date for a Hotel/Airbnb may be different than the start date/time of a Trip due to travel

Reply:
We already have the date of the trip. Can't we use that?

Reply:
I was trying to say that let's say my trip starts tomorrow, April 11, but my flight doesn't leave until 2PM. I get to Italy on April 12 at 4PM, that is a different check in and check out date than when the Trip starts